content/02.introduction.md

```diff
@@ -27,6 +28,10 @@ This is a real-time, collaborative effort that welcomes submissions from scienti
 #### Classification:
 Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses [@isbn:0781760607]. 
 Members of this family include the Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV), which are two species, as well as SARS-CoV-2 [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].
+Coronaviridae are classified into four main groups called alphacoronaviruses, betacoronaviruses, gammacoronaviruses and deltacoronaviruses. 
+Alphacoronaviruses and betacoronaviruses are mainly found in mammals such as bats, rodents, civets, and humans, whereas gammacoronaviruses and deltacoronaviruses have their gene source from birds and are mainly found in birds.
+Phylogenetic analysis of a PCR amplicon fragment from five patients in a family cluster who visited Wuhan, China along with the total virus genome of 29.8 kilobases indicates that the virus is a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus.
+The sarbecovirus lineage also includes the human SARS coronavirus (@doi: 10.1016/S0140-6736(20)30154-9). 
```

Comment:
```suggestion
The sarbecovirus lineage also includes the human SARS coronavirus [@doi:10/ggjs7j]. 
```
Manubot doesn't like DOIs that have parentheses, so I converted this using shortdoi.org (in case you're wondering!)